packages/sit-onyx/src/components/OnyxButton/OnyxButton.stories.ts

```diff
@@ -0,0 +1,67 @@
+import { defineStorybookActionsAndVModels } from "@sit-onyx/storybook-utils";
+import type { Meta, StoryObj } from "@storybook/vue3";
+import OnyxButton from "./OnyxButton.vue";
+import { BUTTON_COLORS, BUTTON_TYPES, BUTTON_VARIANTS } from "./types";
+
+/**
+ * Buttons serve as fundamental components in UI design,
+ * acting as gateways for user interactions and pivotal points for
+ * initiating actions within an interface.
+ * Whether prompting users to submit forms,
+ * navigate through pages, or trigger specific functionalities,
+ * buttons play a pivotal role in guiding users through their
+ * digital journey.
+ */
+const meta: Meta<typeof OnyxButton> = {
+  title: "components/OnyxButton",
+  ...defineStorybookActionsAndVModels({
+    component: OnyxButton,
+    events: ["click"],
+    argTypes: {
+      type: {
+        options: BUTTON_TYPES,
+        control: { type: "select" },
+      },
+      color: {
+        options: BUTTON_COLORS,
+        control: { type: "select" },
+      },
+      variant: {
+        options: BUTTON_VARIANTS,
+        control: { type: "select" },
+      },
```

Comment:
```suggestion
      type: {
        options: BUTTON_TYPES,
      },
      color: {
        options: BUTTON_COLORS,
      },
      variant: {
        options: BUTTON_VARIANTS,
      },
```

Type select is set automatically when options are defined :)